### PR TITLE
Document corresponding jscs rules

### PR DESCRIPTION
**Pull request by hzoo**
Ok dang that took a while.

Ref https://github.com/airbnb/javascript/pull/608#issuecomment-165879126. @ljharb 

cc @markelog (probably missed some)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -292,7 +302,9 @@ Other Style Guides
   > Why? In general we consider it subjectively easier to read. It improves syntax highlighting, and is also more easily optimized by many JS engines.
 
   eslint rules: [`quote-props`](http://eslint.org/docs/rules/quote-props.html).
-
+  
+  jscs rules: [`disallowQuotedKeysInObjects: "allExcept": ["reserved"]`](http://jscs.info/rule/disallowQuotedKeysInObjects).
```
Comment (hzoo) [inline README.md]: Don't know if we want to but the actual value or just a link is good.

Comment (ljharb) [inline README.md]: just the link i think should be fine.

**Comment by kesne** [comment]
I wonder if it might look a little cleaner to put the jscs rules and eslint rules on the same line?

**Comment by hzoo** [comment]
Yeah I did that at first but I wasn't sure

**Comment by goatslacker** [comment]
two lines is ok.

**Comment by ljharb** [comment]
Thanks!

**Pull request merged**